app/views/user_mailer/_due_today.html.erb

```diff
@@ -1 +1 @@
-<p><%= simple_format(replace_variables(@app_configs.upcoming_checkin_email_body)) %></p>
\ No newline at end of file
+<%= render html: simple_format(replace_variables(@app_configs.upcoming_checkin_email_body)).gsub('\n', '').html_safe %>

```

Comment:
The `gsub` is necessary because `simple_format` converts `\n`s to html linebreaks but does not remove them. I'm not sure how to handle it otherwise, unless there's another way of storing linebreaks in our defaults.